addon/README.md

```diff
@@ -0,0 +1,17 @@
+This is the root of the add-on.  This add-on is an [Embedded Extension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Embedded_WebExtensions), which means we have a [Bootstrap Extension](https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions) that wraps a [WebExtension](https://developer.mozilla.org/en-US/Add-ons/WebExtensions).
```

Comment:
Nit: If you remove those `/en-US/` fragments, it would automatically redirect users to their browsers' language preference (and fallback to /en-US/).